oat-sa/extension-tao-itemqti-pci
# fix: use responseDeclaration cardinality to shape response

related to https://oat-sa.atlassian.net/browse/LSI-1548

### Description 

Legacy MathEntry PCI have cardinality: multiple in gap mode. New mathEntry pci runtime shapes single cardinality response, this leads to error 500 on backend when such response cannot be handled by backend

Changes introduced use responseDeclaration to shape the response to fit it.

### How to test

Use [branch](https://github.com/oat-sa/tao-item-runner-qti-fe/tree/fix/LSI-1548/forward-cardinality-to-pci) from this PR :  https://github.com/oat-sa/tao-item-runner-qti-fe/pull/349
 - run the script `php index.php '\oat\taoQtiItem\scripts\initEventCreatorLoad'`
 - create file `config/taoQtiItem/debug_portable_element.conf.php` with contents: 
 ```
 <?php
return [
        //'pciIdentifier' => 'path/to/pci/source/files',
        'mathEntryInteraction' => 'qtiItemPci/views/js/pciCreator/ims/mathEntryInteraction'
        //'textReaderInteraction' => 'pciSamples/views/js/pciCreator/dev/textReaderInteraction/'
    ];
?>
 ```
 - enter item authoring, preview and check the response submitted
